sterling/account.py

```diff
@@ -1,14 +1,14 @@
 import json
 from urllib.parse import urljoin
 from url import url as BASE_URL
-from sterling.Sterling import  Sterling
+from sterling.Sterling import Sterling
 from sterling.utils.calls import calls
 
 
 class Account(Sterling):
     def InterbankTransferReq(self, body):
-        url = urljoin(BASE_URL(self.url), "/sterling/accountapi/api/Spay/InterbankTransferReq")
+        url = urljoin(BASE_URL(self.url),
+                      "/sterling/accountapi/api/Spay/InterbankTransferReq")
```

Comment:
Better to make the URL constant as you've done with atlab

Reply:
That would be in another PR